swift-package-baseline/baseline.txt

```diff
@@ -0,0 +1 @@
+v2.4.0-rc.1
```

Comment:
For this baseline, there should be a baseline file `swift-package-baseline/8284de2e35ce98b414d317f00eed26988d7bbbae/MapboxDirections.json` as `swift-package-baseline/2b14b8c276523a01dc730897593990ef83c71320/MapboxDirections.json` is for release `2.3.0`.

Reply:
nice catch, fixed